modules/decision_reviews/app/sidekiq/decision_reviews/failure_notification_email_job.rb

```diff
@@ -51,29 +45,38 @@ def perform_form_and_evidence
 
     def perform_all
       enabled? &&
-        secondary_forms_enabled? &&
         (submissions.present? || submission_uploads.present? || errored_secondary_forms.present?)
     end
 
-    def vanotify_service
-      @service ||= ::VaNotify::Service.new(VANOTIFY_API_KEY)
+    def get_callback_config(email_type, appeal_type)
+      case email_type
+      when :form
+        [DecisionReviews::FormNotificationCallback, 'form submission', DecisionReviews::V1::FORM_TEMPLATE_IDS[appeal_type]]
+      when :evidence
+        [DecisionReviews::EvidenceNotificationCallback, 'evidence submission to Lighthouse', DecisionReviews::V1::EVIDENCE_TEMPLATE_IDS[appeal_type]]
+      when :secondary_form
+        [DecisionReviews::EvidenceNotificationCallback, 'secondary form submission to Lighthouse', DecisionReviews::V1::SECONDARY_FORM_TEMPLATE_ID]
+      end
     end
 
-    def vanotify_service_with_callback(submission, template_id)
+    def vanotify_service_with_callback(submission, email_type, reference)
+      appeal_type = submission.type_of_appeal
+      callback_klass, function, email_template_id = get_callback_config(email_type, appeal_type)
+      service_name = DecisionReviews::V1::APPEAL_TYPE_TO_SERVICE_MAP[appeal_type]
+
       callback_options = {
-        callback_klass: DecisionReviews::FormNotificationCallback.to_s,
+        callback_klass: callback_klass.to_s,
         callback_metadata: {
           email_type: :error,
-          service_name: APPEAL_TYPE_TO_SERVICE_MAP[submission.type_of_appeal],
-          function: 'form submission',
+          service_name:,
+          function:,
           submitted_appeal_uuid: submission.submitted_appeal_uuid,
-          email_template_id: template_id
+          email_template_id:,
+          reference:,
```

Comment:
We were previously not passing `reference` in the `callback_metadata`, which is half of the reason it wasn't showing up in our `DecisionReviewNotificationAuditLog` records. The other half of the reason is that we weren't accessing the field correctly in Form/EvidenceNotificationCallback (explained in more detail below)